server/chat-plugins/mafia.ts

```diff
@@ -106,7 +101,9 @@ const VALID_IMAGES = [
 
 let MafiaData: MafiaData = Object.create(null);
 let logs: MafiaLog = {leaderboard: {}, mvps: {}, hosts: {}, plays: {}, leavers: {}};
-let hostBans: MafiaHostBans = Object.create(null);
+
+Punishments.roomPunishmentTypes.set('GAMEBAN', 'banned from playing games');
```

Comment:
i feel like these should probably be namespaced (MAFIAGAMEBAN / MAFIAHOSTBAN), and the description should probably mention that they're mafia games specifically